modules/doc/content/getting_started/examples_and_tutorials/index.md

```diff
@@ -35,10 +37,87 @@ demonstrates and how to use it is provided here:
 - [Example 20: UserObjects](examples/ex20_user_objects.md)
 - [Example 21: Debugging](examples/ex21_debugging.md)
 
+### Module Use Examples
+
+Example problems are provided for some of the individual [modules/index.md]. A comprehensive list of all that are available is given below. Simply expand the example set under a module header and follow their links.
+
+###### Contact
+
+- [modules/contact/TwoDimensionalSphericalIndenterMortar.md]
+- [modules/contact/TwoDimensionalSphericalIndenterNodeFace.md]
+- [modules/contact/BerkovichIndenterNodeFace.md]
```

Comment:
Do you think you could make an example.md file inside each module (that has examples). Then use `!include` command here as well as in the module. That way if another gets added it will show up both places automatically.

Reply:
I'll see what I can do!